test/dotnet.Tests/CommandTests/ToolManifestFinderTests.cs

```diff
@@ -95,8 +95,6 @@ public void GivenManifestFileInDotConfigDirectoryItGetContent()
         }
 
         [Fact]
-        // https://github.com/JamesNK/Newtonsoft.Json/issues/931#issuecomment-224104005
```

Comment:
The new library also has the "last one win" when it comes to duplicate properties.

Reply:
forget to remove. In this case the default system.text.json behavior is exactly what I want

Reply:
cc @bartonjs